### ISSUE TITLE
Better instructional text

### ISSUE THREAD

**Issue by toribr** [cityofaustin/techstack#2769]
Probably the help text should read:

Select existing *published* pages in the order you want them to display within each heading. Pages should be added only once to any single guide.

**Comment by desigonz** [comment]
@toribr, can you clarify where this microcopy lives?
Assigning to @srigdon to quickly review the copy.

**Comment by toribr** [comment]
The copy should be in Joplin on the guide template. It's not, though. Here's my version of the guide template:
![1851 Guide](https://user-images.githubusercontent.com/3790067/64353999-3dddda80-cfc4-11e9-8998-937e19005d35.png)

**Comment by desigonz** [comment]
This section in particular: 
![image](https://user-images.githubusercontent.com/5839508/64354779-b09b8580-cfc5-11e9-96b4-991d98eee568.png)

**Comment by srigdon** [comment]
Hm, I'd like to review the question mark helper text, too--should that be a separate issue, @desigonz? 

For that section in particular, let's try the following copy changes: 

Add a section header and pages to each section. > Change to: 
Add a section heading and pages to each section. 

Select existing pages in the order you want them to display within each heading. Pages should be added only once to any single guide. > Change to: 
Select published or live pages in the order you want them to display under this heading. Use the up and down arrows on the right to change the order. Only add each page once to any single guide. 

Also, not sure if this is captured elsewhere but I think that CHOOSE ANOTHER PAGE button should be CHOOSE A DIFFERENT PAGE. As is, it competes with the ADD ANOTHER button below it for the same meaning or function. 

Also, when you choose a page, all pages, drafts and live, appear in the list. Can we just display live pages here? 

@toribr, your "Note: need clarification on outcomes wrt published pages being added/removed to guides either intentionally or by unpublishing." above, did you get an answer on this?

**Comment by toribr** [comment]
@srigdon @desigonz 

No, we haven't addressed this. Since we only allow published pages to be included (at this time), what we'd need would be for an alert to take place **on any page** that is included in a guide. 

user: clicks unpublish on a page, gets notification message, proceeds or not
If user proceeds, system sends message (what kind of message? email? ) to guide owner that a page in "name of guide" has been unpublished by {name of person}.

Lots to think over.

Something like: Unpublishing this page will remove it from:
•  "guide name"
Are you sure you want to unpublish this page? 

It gets more complicated. What if the page is owned by someone other than the guide owner? Is the guide owner notified that the page has been removed? What recourse does the guide owner then have?

**Comment by desigonz** [comment]
@toribr Thanks for clarifying all this! I've marked this as sprint 40—since it's a lot to think over and we have tons on our plate for this sprint, let's review then.
